### DISCUSSION TITLE
Launch the server successfully, but the client cannot reach the server port

### DISCUSSION THREAD

**Discussion by heiseon** [alibaba/GraphScope#387] [Q&A]
I launch the server following by official docs (scripts/prepare_env.sh) successfully.
But in the other host which install client environment cannot connect to the server port.

How do I find server port.
In server host I use " kind get kubeconfig" , I can hold the ip and port.
where am I wrong?

**Environment (please complete the following information):**
 - GraphScope version: v0.4.1
 - OS: Red hat
 - Version 7
 - Kubernetes Version latest

**Comment by acezen** [comment]
seems is not an issue, move to discussion.

**Comment by acezen** [comment]
hi @heiseon, I think you need to copy the kube config from the host you run prepare_env.sh to the cient host. the client host need to know the kubernetes cluster configuration.

you can run `kubectl get nodes` in client host to check client  can access to kubernetes cluster.

**Reply by heiseon** [reply]
you mean client host should install kubectl?

My problem is when I execute "kind get kubeconfig"  the port is 35704, which is dynamic port. 
I execute "kind get kubeconfig --internal"  the port is 6344, which is static port. 
when I telnet in client host : telnet <ip>:35704， connection is refused.

FInally I wanna ask is there any DingDing chatting Group for GraphScope?
Thanks a lot.

**Reply by neblen** [reply]
```
export NODE_IP=$(kubectl --namespace graph get pod -o jsonpath="{.status.hostIP}" graphscope-cordinator)
export NODE_PORT=$(kubectl --namespace graph get services -o jsonpath="{.spec.ports[0].nodePot}" graphscope-coordinator-service)
echo "GraphScope service listen on ${NODE_IP}:${NODE_PORT}"
```
I think you can use these commands to find the ip and port.

**Reply by acezen** [reply]
the `prepare_env.sh` script is to help user deploy a kubernetes cluster in local. So we suggest you install graphscope client  in the same host. 

Btw, we do have a DingDing chatting Group, the group id is 31533139, welcome to join the Group chat.
